checks.d/docker_daemon.py

```diff
@@ -250,8 +251,11 @@ def check(self, instance):
         # containers running with custom cgroups?
         custom_cgroups = _is_affirmative(instance.get('custom_cgroups', False))
 
+        # submit healtcheck service checks?
+        health_service_checks = _is_affirmative(instance.get('health_service_checks', False))
```

Comment:
I think we don't need this parameter anymore. If the list is there, enable health checks, otherwise disable them. So you can look for the white list param directly, maybe check if it isn't empty.